src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -785,6 +777,10 @@ def set_prometheus_access_info(self, user: str, password: str) -> OrchResult[str
         """set prometheus access information"""
         raise NotImplementedError()
 
+    def generate_certificates(self, module_name: str) -> OrchResult[Optional[Dict[str, str]]]:
+        """set prometheus access information"""
```

Comment:
comment is for a different function it seems

Reply:
Fixed